.gitignore

```diff
@@ -35,7 +35,6 @@ yarn-error.log
 ###< liip/imagine-bundle ###
 
 .idea/
-.vscode/
```

Comment:
This needs to be setup differently (e.g. like for `bin` folder below).

The point is to allow the two files you added, but disallow everything else, e.g:

```
.vscode/*
!.vscode/extensions.json
!.vscode/settings.json
```

Reply:
This should be done now.